### PR TITLE
update format of a tcp ServiceEntry

### PR DESCRIPTION
**Pull request by vadimeisenbergibm**
remove discovery, add hosts/addresses.

### CODE REVIEW

File: _docs/tasks/traffic-management-v1alpha3/egress-tcp.md [hunk 1]
```diff
@@ -44,16 +44,17 @@ metadata:
...
+    - 91.198.174.192/27
+    - 103.102.166.224/27
+    - 198.35.26.96/27
+    - 208.80.153.224/27
+    - 208.80.154.224/27
   ports:
-  - number: 443
-    name: tcp
-    protocol: TCP
-  discovery: NONE
```
Comment (rshriram) [inline _docs/tasks/traffic-management-v1alpha3/egress-tcp.md]: this should be resolution: NONE for clarity

Comment (vadimeisenbergibm) [reply _docs/tasks/traffic-management-v1alpha3/egress-tcp.md]: fixed

**Pull request merged**